build/committee/1307016/contributions/index.json

```diff
@@ -71,17 +71,17 @@
   },
   {
     "Filer_ID": "1307016",
-    "Tran_Amt1": 9499.5,
+    "Tran_Amt1": 5500.5,
     "Tran_Date": "2016-09-21",
     "Tran_NamF": null,
-    "Tran_NamL": "CP VI Franklin, LLC"
+    "Tran_NamL": "CP V JLS, LLC"
   },
   {
     "Filer_ID": "1307016",
-    "Tran_Amt1": 5500.5,
+    "Tran_Amt1": 9499.5,
     "Tran_Date": "2016-09-21",
     "Tran_NamF": null,
-    "Tran_NamL": "CP V JLS, LLC"
+    "Tran_NamL": "CP VI Franklin, LLC"
```

Comment:
I keep seeing this diff locally too but for the life of me cannot figure out why this is happening. There is a sort condition on `Tran_NamL` for crying out loud!